modulefiles/build.wcoss_dell_p3

```diff
@@ -29,3 +29,6 @@ module use /gpfs/dell2/emc/modeling/noscrub/emc.nemspara/soft/modulefiles
 module load esmf/8.0.0_ParallelNetCDF
 
 export WGRIB2_ROOT=/gpfs/dell2/emc/modeling/noscrub/George.Gayno/wgrib2
```

Comment:
Could loading the grib_util modules (which contains $WGRIB2) replace this line or is this a special wgrib2 you're testing?

Reply:
This is not the wgrib2 program.  It is the library API version of wgrib2 that is callable from Fortran.  We hope to formally implement it shortly (it is required for GFS v16).

Reply:
Got it, thanks!